From 1756e8988328036393de44cf20571b6513fa1143 Mon Sep 17 00:00:00 2001
From: nordhausen <stefan.nordhausen@axiros.com>
Date: Fri, 28 Feb 2014 14:22:20 +0100
Subject: [PATCH] Adapt IPV4_RESERVED according to RFC 5735 Appendix A

---
 netaddr/ip/__init__.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/netaddr/ip/__init__.py b/netaddr/ip/__init__.py
index 7a93510b7f9fde1bd94757c50f9e7b0ac6463fbf..200753d9e73ecfad530044ce30e166b17d63b81c 100644
--- a/netaddr/ip/__init__.py
+++ b/netaddr/ip/__init__.py
@@ -1918,10 +1918,7 @@ IPV4_MULTICAST = IPNetwork('224.0.0.0/4')
 IPV4_6TO4 = IPNetwork('192.88.99.0/24')    #   6to4 Relay Anycast
 
 IPV4_RESERVED = (
-    IPNetwork('128.0.0.0/16'),      #   Reserved but subject to allocation
-    IPNetwork('191.255.0.0/16'),    #   Reserved but subject to allocation
     IPNetwork('192.0.0.0/24'),      #   Reserved but subject to allocation
-    IPNetwork('223.255.255.0/24'),  #   Reserved but subject to allocation
     IPNetwork('240.0.0.0/4'),       #   Reserved for Future Use
 
     #   Reserved multicast
-- 
2.9.5

